homeassistant/components/image_processing/__init__.py

```diff
@@ -1,11 +1,14 @@
 """Provides functionality to interact with image processing services."""
+from __future__ import annotations
+
 import asyncio
 from datetime import timedelta
 import logging
-from typing import Final, TypedDict, final
+from typing import Any, Final, TypedDict, final
 
 import voluptuous as vol
 
+from homeassistant.components.camera import Image
```

Comment:
If we don't have to, I think we should move imports from other components into `if TYPE_CHECKING`. That will proactively help prevent circular imports between base components.

Reply:
I disagree. We should only do that when needed. Normally there should not be any circular imports if our design is correct.

Keeping it simple will reduce burden on contributors and members.